### ISSUE TITLE
CORTEX-S v11 compiled explicit-FP32 CE — zero-credit systems prereg

### ISSUE THREAD

**Issue by vinceackermann2-sys** [vinceackermann2-sys/tam-research#910]
## Classification
`ZERO_CREDIT_COMPILED_EXPLICIT_FP32_CE_SYSTEMS_OPTIMIZATION_ONLY`

This preregistration follows the consumed v10 attempt #906 and its authoritative durable readout #909. It authorizes only source inspection, an isolated fresh branch, CPU/static implementation/tests, ordinary original CI, and merge if all original checks pass. It authorizes **no Modal invocation, no GPU/H100 allocation, no engineering-seed reservation/consumption, no seed 8100, no scientific seeds, no full 2B training, and no scientific/breakthrough claim**.

## Authoritative antecedent
v10 is permanently consumed and MUST NOT be rerun/retried/reused:
- issue #906
- workflow `34682658761`, attempt 1
- job `103524034151`
- source `6e2b535a2eb8fdea6579fc27849091ee5125b9ea`
- tree `2a019c7d5785e9c4b7e48eda15c9b6faec30e8f3`
- harness `48c5d5a2ee7ff3688785f8e7943d537e5215358b`
- engineering seed `2026091012`
- result root `/vol/cortex-s-v0/100m-2b/preflight-v10-autocast-ce-no-explicit-fp32`
- durable result: `SYSTEMS_PREFLIGHT_ERROR`
- classification: `ENGINEERING_SYSTEMS_PREFLIGHT_V10_AUTOCAST_CE_NO_EXPLICIT_FP32_ERROR`
- error: `RuntimeError('CUDA autocast CE no longer matches explicit-FP32 reference')`
- H100 allocation started; no valid calibration/throughput/compile/projection metrics exist.

The read-only recovery #909 proved `RESULT.json` exists and preserved the consumed state without GPU/new seed/Volume mutation.

## Frozen live boundary
Immediately before this preregistration live `main` is:
- SHA `52fbaeaefccd8d69192ebcba80b0c54e35584d1c`
- tree `3667b53fee1aca7b3367373b1a17186beba74d04`

Frozen production trainer:
- `architectures/cortex_s/experiments/scale100m_2b/train.py`
- blob remains `004b66b549d3d64f2dde7614ec84f22b9f37a7c6`
- SHA256 `d0d91de09bba8b7631913b3e9c0407ab2bb0114f7bba133c123ee5598d490461`

Production currently compiles only `runner(x)` via `_compile_model`; its explicit-FP32 CE remains outside that compiled model call:
`F.cross_entropy(logits.float().reshape(-1, logits.size(-1)), y.reshape(-1))`.

## Frozen v11 hypothesis
Preserve the existing explicit-FP32 loss expression exactly, but compile the **loss-only** function separately with `torch.compile(..., mode="max-autotune-no-cudagraphs", fullgraph=True)`.

The systems candidate may replace only the optimizer-step realization so that:
1. model/logits remain the existing production compiled path;
2. loss inputs remain the exact same BF16 logits and targets;
3. candidate loss source still performs `logits.float()` before `F.cross_entropy`, preserving the explicit-FP32 reference semantics that v10 violated;
4. Inductor is given an isolated fullgraph cast+CE region and may fuse/eliminate intermediate realization if supported;
5. gradient accumulation, clipping, optimizer, LR, data, model, routing, recurrence, scan/MoE/attention, and single bundled host readback remain unchanged.

No Liger dependency or eager Dynamo boundary is permitted in v11. This is not a retry of v9 and must not modify `fused_linear_ce_v1.py`.

## Zero-credit acceptance gates
Before any paid authority can exist, the fresh candidate/tests must prove:
- literal explicit-FP32 CE formula and target/reduction semantics are unchanged;
- deterministic tiny-CORTEX loss and **every parameter gradient** match the production explicit-FP32 reference under frozen tight CPU tolerances;
- the loss-only compiler is exactly `max-autotune-no-cudagraphs`, `fullgraph=True`;
- production `_compile_model` and trainer remain untouched;
- two microsteps / 65,536 tokens per optimizer step remain unchanged;
- v6 single bundled device→CPU loss+safety readback remains exactly one and before optimizer update;
- non-finite loss/gradient still aborts before optimizer update;
- builder restoration is exact;
- no Liger import/path, no Modal import/invocation, no GPU trigger, no `train_full_2b` call, no seed 8100/scientific seeds, and no paid identifier exists in this candidate stage;
- full existing CI, CORTEX-S CPU gate, and 100M/2B zero-credit gate pass on the exact PR head using only their original automatic attempts.

## Fresh zero-credit identity
- branch: `research/cortex-s-v11-compiled-explicit-fp32-ce-v1`
- candidate: `architectures/cortex_s/compiled_explicit_fp32_ce_v1.py`
- tests: `architectures/cortex_s/tests/test_compiled_explicit_fp32_ce_v1.py`

No engineering seed, future paid trigger title, phase, or result namespace is reserved here.

## Progression rule
If the zero-credit candidate merges after a fresh race/blob audit, a later separately frozen source-bound H100 preflight may reserve fresh one-shot identifiers and must retain the same systems gate as v6-v10: exact `101,778,112` params; grouped-v4 + validated Triton 24/24 and 0 fallback; BF16, fused AdamW, seq512/microbatch64/grad-accum2; 5 warmup + 40 measured optimizer steps / 2,621,440 tokens; <=70 GiB; projection `2_000_000_000 / measured_tps * 1.10 + compile_seconds <= 8500`.

Even a systems PASS would only authorize a separately preregistered full-training stage. No full 2B run is authorized by this issue.
